docs/docs/how-tos/nebari-aws.md

```diff
@@ -0,0 +1,132 @@
+
+```bash
+nebari init aws --project projectname --domain domain --auth-provider password
+```
+:::note Note
+You will be prompted to enter values for some of the choices above if they are omitted as command line arguments (for example project name and domain)
+:::
+
+Once `nebari init` is executed, you should then be able to see the following output:
+```bash
+Securely generated default random password=*** for Keycloak root user
+stored at path=/tmp/NEBARI_DEFAULT_PASSWORD
+INFO:botocore.credentials:Found credentials in environment variables.
+```
+:::tip
+MacOS generates a programmatic directory stored in `/private/var` and defines the `$TMPDIR` environment variable for locating the system temporary folder. We advise the user to look for the `NEBARI_DEFAULT_PASSWORD` file in the following  `/var/folders/xx/xxxxx/T` path.
+:::
+
+You can see that Nebari is generating a random password for the root user of Keycloak. This password is stored in a temporary file and will be used to authenticate to the Keycloak server once Nebari's infrastructure is fully deployed.
+
+The generated `nebari-config.yaml` is the configuration file that will determine how the cloud infrastructure and Nebari is built and deployed in the next step. But at this point it’s just a text file. You could edit it manually if you are unhappy with the choices, or delete it and start over again.
+
+For additional information about the `nebari-config.yaml` file and extra flags that allow you to configure the initialization , see the [Understanding the nebari-config.yaml file](/tutorials/overview.md) documentation.
+
+## Deploying Nebari
+
+Finally, with the `nebari-config.yaml` created, Nebari can be deployed for the first time:
+
+```bash
+nebari deploy -c nebari-config.yaml
+```
+The terminal will prompt you to press `[enter]` to check auth credentials, which were added in the previous step by the initialization command. Once authenticated, the infrastructure deployment process will start and might take around a couple minutes to complete.
+
+Once you reach stage `04-kubernetes-ingress` you will be prompted to set the **A/CNAME** records manually for your registered domain name. Please follow the instructions in the [Nebari DNS](/how-tos/domain-registry.md) section for more information regarding the domain `A/CNAME` records and how to automatically generate them.
+
+If the deployment is successful, you will see the following output:
+```bash
+[terraform]: Nebari deployed successfully
+Services:
+ - argo-workflows -> https://projectname.domain/argo/
+ - conda_store -> https://projectname.domain/conda-store/
+ - dask_gateway -> https://projectname.domain/gateway/
+ - jupyterhub -> https://projectname.domain/
+ - keycloak -> https://projectname.domain/auth/
+ - monitoring -> https://projectname.domain/monitoring/
+Kubernetes kubeconfig located at file:///tmp/NEBARI_KUBECONFIG
```

Comment:
@bskinn yes it will be cleaned up, but nebari re-generates the file for every run, so we don't have any problems. I agree that this question is important, but this should be added to troubleshooting as `Can't find NEBARI_KUBECONFIG file on /tmp, what should I do?` or a similar question, right?

Reply:
This issue will be addressed in [here](https://github.com/nebari-dev/nebari/issues/83#issuecomment-1171641391)